src/internal/verbose.go

```diff
@@ -0,0 +1,14 @@
+package yey
+
+import (
+	"fmt"
+	"os"
+)
+
+var ()
```

Comment:
I would co-localize the IsVerbose variable here. It makes more sense to put variables in their context than in a list of declaratations. 

Congrats on logging changes it’s much better this way!